Fakebook.Profile/Fakebook.Profile.RestApi/Startup.cs

```diff
@@ -17,36 +19,92 @@ namespace Fakebook.Profile.RestApi
 {
     public class Startup
     {
-        public Startup(IConfiguration configuration) {
+        public Startup(IConfiguration configuration, IWebHostEnvironment env)
+        {
             Configuration = configuration;
+            _env = env;
         }
 
         public IConfiguration Configuration { get; }
+        private readonly IWebHostEnvironment _env;
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services) {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddCors(options =>
+            {
+                options.AddDefaultPolicy(
+                    builder =>
+                    {
+                        builder.WithOrigins("https://fakebook.revaturelabs.com/")
+                            .AllowAnyMethod()
+                            .AllowAnyHeader()
+                            .AllowCredentials();
+                    });
+            });
+
+            services.AddAuthentication(
+                JwtBearerDefaults.AuthenticationScheme)
+                .AddJwtBearer(options =>
+                {
+                    options.Authority = "https://dev-7862904.okta.com/oauth2/default";
+                    options.Audience = "api://default";
+
+                    // Won't send details outside of dev env
+                    if (_env.IsDevelopment())
+                    {
+                        options.IncludeErrorDetails = true;
+                    }
+                });
 
             services.AddControllers();
-            services.AddSwaggerGen(c => {
+
+            services.AddSwaggerGen(c =>
+            {
                 c.SwaggerDoc("v1", new OpenApiInfo { Title = "Fakebook.ProfileRestApi", Version = "v1" });
             });
+
+            // for the profile db
+            services.AddDbContext<ProfileDbContext>(options
+                => options.UseNpgsql(Configuration.GetConnectionString("FakebookProfile")));
```

Comment:
if this is where you're looking for the connection string, then correct the `kubernetes_deployments.yaml` file on line 32 to `ConnectionStrings__FakebookProfile`. also, provide a dummy value in `appsettings.json` so the migrations script can be generated in the pipeline.